src/renderer/components/Message.js

```diff
@@ -54,7 +54,7 @@ class RenderMessage extends React.Component {
 
     const contact = {
       onSendMessage: () => console.log('send a message to', fromId),
-      onClick: () => console.log('clicking contact', fromId)
+      onClick: () => { console.log('click contact') }
```

Comment:
Is this unfinished? cc @Jikstra

Reply:
Afaict, this `contact` is passed into `<Message>` but ends up in `renderEmbeddedContact()` (in `conversations/`) but that method doesn't seem to be used anywhere.